### DISCUSSION TITLE
Export the parent reconstructed beta grains and orientations (.CTF file)

### DISCUSSION THREAD

**Discussion by Naga-Chaithanya** [mtex-toolbox/mtex#1618] [Ask Anything] [answered]
Hello Everyone.
I have used the method described in the link "https://mtex-toolbox.github.io/TiBetaReconstruction.html" for the Parent Beta Phase Reconstruction in Titanium Alloys. The method works very well and gives the parent grains and orientations, I can plot them in MTEX after reconstruction. 
  **I want to export this reconstructed beta phase file as a separate dataset** (.CTF or .ANG) after following the above steps to directly import beta phase later for other calculations. The beta phase orientations should be in "parentEBSD". when I export that file and open this, it again shows the original alpha orientations rather than beta orientations. (please see the image below)

Could you suggest a method to export the reconstructed beta data file (.CTF)?
![image](https://user-images.githubusercontent.com/40796258/198567611-2298f37e-eef4-49ce-a1fc-2661a4f086eb.png)

**Comment by ralfHielscher** [comment, marked as answer]
This is quite strange as you did everything correctly. Are you sure you imported the right file and assigned it to variable?

After all a ctf file is simply a text file. So you can inspect the phase of the data.

Ralf.

**Reply by Naga-Chaithanya** [reply]
Thank you for the suggestion Prof. Ralf Hielscher,
  I have interchanged the phaseID of alpha with beta. I could get the parent grains as shown below. 

Only the orientations (euler angles) within the beta grains are varying too much like each grains has its different variants but beta phase should not have any. I remember one of the codes used is of "job.parentGrains.meanOrientation" to average out the parent orientations. yet the parentEBSD has many variants. Is it possible to export the CTF file containing these mean orientation rather than these "IPF beta map shown below with more orientation gradients"  
![image](https://user-images.githubusercontent.com/40796258/198590313-879e590a-aedb-4ff7-b5e2-487bd7bd4f90.png)
